gigascience/gigadb-website
# New feature/1556 fix faq page tabpanel misuse

# Pull request for issue: #1556

This is a pull request for the following functionalities:

* http://gigadb.gigasciencejournal.com:9170/site/faq remove tablist and use accordion pattern instead

## How to test?

* run playwright tests
```
cd playwright
npm i
npm test
```
* Navigate to http://gigadb.gigasciencejournal.com:9170/site/faq
* Run WAVE audit and verify no "errors" reported


## How have functionalities been implemented?

* Removed all tab-related roles
* Updated FAQ titles to be wrapped in a button instead of link, and fixed styles accordingly

## Any issues with implementation?

Bootstrap 3 is a bit irregular in terms of accessibility, the collapsible panels get an aria-expanded attribute that should not be there. More modern versions of Bootstrap do not have this issue

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
